src/core/textarea/index.tsx

```diff
@@ -14,9 +26,12 @@ const Component = React.memo((props: ITextareaProps) => {
         name={value}
         value={value}
         rows={4}
-        className="w-full rounded-lg border border-gray-300 p-2 focus:border-transparent focus:outline-none focus:ring-2 focus:ring-blue"
+        className={inputClassNames}
         onChange={(e) => (onChange ? onChange(e.target.value) : null)}
       />
+      {errorMessage && (
```

Comment:
{errorMessage && (
          <div className="mt-2 text-sm text-red">{errorMessage}</div>
        )}

to 

{errorMessage ?  (
          <div className="mt-2 text-sm text-red">{errorMessage}</div>
        ): null}